### ISSUE TITLE
Handling ORCID data after user leaves Penn State

### ISSUE THREAD

**Issue by EricDurante** [psu-libraries/researcher-metadata#68]
When the MDDB app writes data to a person's ORCID record on their behalf, the person is then unable to modify that data in their ORCID record via the ORCID website. They can go to the ORCID website and delete it if they want, but they can't update it. Only the app that created the data can modify it. This is particularly relevant for data that would need to change at the same time that a person loses the ability to change it. For example, if a person has posted their Penn State employment info from the MDDB app for their current position, but then they leave the university for another job, they would need to update that employment info in ORCID to indicate the end date of their old position, but they wouldn't be able to do that since they'd no longer have access to the MDDB app. In this case, we would need to give MDDB admins the ability to push updated data to ORCID for that user if they request such a change after they have lost access themselves.

**Comment by anaelizabethenriquez** [comment]
@ajkiessl Is this still a live issue? If so, it seems pretty important now that researchers need ORCID for more and more things (publishers and funders are requiring its use).

**Comment by ajkiessl** [comment]
@anaelizabethenriquez Yeah this is still an issue, at least on our end.  Once a record has been sent from RMD to ORCID, it cannot be sent again.  I'm curious if ORCID has made any changes on their end to allow editing of records written from external applications, or if there is a way to set edit privileges while sending data to ORCID via the API.  We can have someone investigate this in the next development cycle.
